app/src/main/res/values/strings.xml

```diff
@@ -237,4 +237,7 @@ When using eRouška, nobody can collect your location information or other data
     <string name="no_verification_code">Nemám ověřovací kód</string>
     <string name="no_verification_body">Máte pozitivní test a nepřišla vám SMS pro eRoušku? Napište si o náhradní kód na %1$s.</string>
     <string name="no_verification_caption">Zajímá vás, jak odesílání kódu funguje?\n\nV případě, že máte pozitivní výsledek testu na COVID-19, měl by vám přijít výsledek testu v SMS od testovací laboratoře. Ta výsledky současně odesílá do centrálního informačního systému Ministerstva zdravotnictví.\n\nPoté, co se informace z informačního systému propíšou do systému hygieny, se odesílají automatické SMS eRoušky s ověřovacími kódy.\n\nPokud se výsledky z laboratoře nestihnou propsat do systému hygieny do 22:00, může vám SMS s ověřovacím kódem přijít až následující den.\n\nPokud vám SMS eRoušky nepřišla ani následující den poté, co vám přišla SMS s pozitivními výsledky z laboratoře, napište nám na %1$s.</string>
+
+    <string name="send_data_code_invalid_header">Zadaný kód není platný</string>
```

Comment:
Proč přidáváš české výrazy do angličtiny? Angličtina se přepíše exportem z OneSky a tohle zmizí.

Reply:
Anglický překlad není zatím k dispozici. A angličtina je defaultní a musí být zadána.
Dohodli jsme se, že překlady budeme řešit specificky až budou EFGS změny hotové.
Stejným způsobem jsou takto zadány stringy z EFGS PR.

Reply:
No překlad už běží, takže tahle změna bude přemáznuta tím z OneSky. 

Obecně se mi nelíbí dávat české stringy do angličtiny, ani dočasně, dával bych tam buď překlad od developera, nebo nějaký placeholder (např. `TODO: translate`)

Reply:
Rozjel jsem o tom diskuzi na Slacku v Android kanálu